apps/web/src/common/modules/navigations/gnb/modules/gnb-recent-favorite/modules/GNBFavorite.vue

```diff
@@ -259,6 +363,10 @@ export default {
                 store.dispatch('favorite/load', FAVORITE_TYPE.PROJECT),
                 store.dispatch('favorite/load', FAVORITE_TYPE.PROJECT_GROUP),
                 store.dispatch('favorite/load', FAVORITE_TYPE.CLOUD_SERVICE),
+                store.dispatch('favorite/load', FAVORITE_TYPE.DASHBOARD),
+                store.dispatch('favorite/load', FAVORITE_TYPE.COST_ANALYSIS),
+                getAllCostQuerySetList(),
+                // CAUTION: If GNBDashboardMenu is deprecated, you need to add a request to receive a dashboard list here.
```

Comment:
nitpick: in this case, TODO comment header is more understandable and easy to find by searching.

Reply:
I agree with you